src/ObjectGraph.es6.js

```diff
@@ -33,6 +33,7 @@ function ObjectGraph(opts) {
 
 ObjectGraph.prototype.init = function(opts) {
   this.q = new TaskQueue(opts);
+  this.instanceQueue = new TaskQueue();
```

Comment:
Instances aren't a concept that `object-graph-js` should concern itself with. Clients that need to make this distinction should model this themselves.